docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+
+services:
+  gateway-server:
+    build: .
+    volumes:
+      - .env:/app/.env
+    ports:
+      - "${HTTP_SERVER_PORT}:${HTTP_SERVER_PORT}"
+    env_file:
+      - ./.env
+    networks:
+      - bridged_network
+    depends_on:
+      - postgres
+    restart: on-failure
+    logging:
+      driver: json-file
+      options:
+        max-size: 10m
+
+  # this postgres database is only to be used for testing. It should not be used in production systems
```

Comment:
I kind of like keeping the docker-compose.yml .gitignored so that people can choose their own local configuration without having it automatically added to the repo.